sdk/communication/Azure.Communication.Sms/tests/SmsClientLiveTests.cs

```diff
@@ -197,6 +197,25 @@ public async Task SendingTwoSmsMessages()
             }
         }
 
+        [Test]
+        public async Task SendingSmsFromNullNumberShouldThrow()
+        {
+            SmsClient client = CreateSmsClient();
+            try
+            {
+                SmsSendResult result = await client.SendAsync(
+                   from: null,
+                   to: TestEnvironment.ToPhoneNumber,
+                   message: "Hi");
+            }
+            catch (ArgumentNullException ex)
+            {
+                Assert.AreEqual("from", ex.ParamName);
+                return;
+            }
+            Assert.Fail("SendAsync should have thrown an exception.");
+        }
+
         public void assertHappyPath(SmsSendResult sendResult)
```

Comment:
Unrelated to this PR, but `assertHappyPath` is camelCase instead of PascalCasing.